go/ql/lib/change-notes/2024-06-08-refactor-go-queries-to-use-threatmodelflowsource.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The `Customizations` libraries for several queries, as well as queries which do not have `Customizations` libraries, have been modified to use `ThreatModelFlowSource` to define their `isSource` predicates instead of `RemoteFlowSource`. This means these queries will now respect threat model configurations.
```

Comment:
- I don't think it's worth distinguishing between queries with customization libraries and those without. Most users won't know the difference, and those that do can easily verify that both have been updated.
- I think it's worth explaining briefly that `ThreatModelFlowSource` will be the same as `RemoteFlowSource` except when the query is run in the presence of a specified threat model.